llvm/lib/FuzzMutate/FuzzerCLI.cpp

```diff
@@ -107,7 +108,24 @@ void llvm::handleExecNameEncodedOptimizerOpts(StringRef ExecName) {
       Args.push_back("-passes=loop-reduce");
     } else if (Opt == "irce") {
       Args.push_back("-passes=irce");
-
+    } else if (Opt == "dse") {
+      Args.push_back("-passes=dse");
+    } else if (Opt == "loop_idiom") {
+      Args.push_back("-passes=loop-idiom");
+    } else if (Opt == "reassociate") {
+      Args.push_back("-passes=reassociate");
+    } else if (Opt == "lower_matrix_intrinsics") {
+      Args.push_back("-passes=lower-matrix-intrinsics");
+    } else if (Opt == "loop_reroll") {
+      Args.push_back("-passes=loop-reroll");
+    } else if (Opt == "loop_interchange") {
+      Args.push_back("-passes=loop-interchange");
+    } else if (Opt == "loop_fusion") {
+      Args.push_back("-passes=loop-fusion");
```

Comment:
Remove these three, these are non-production passes and at least loop reroll and loop fusion are badly broken (not so sure about interchange).

Reply:
Done